private/buf/bufformat/testdata/proto3/all/v1/all.golden.proto

```diff
@@ -0,0 +1,117 @@
+// top-level comment 1
+// top-level comment 2
+
+syntax = "proto3"; // syntax-level-inline comment
+
+// between-syntax-and-package comment
+package all.v1; // package-level-inline comment
+
+import "custom.proto";
+// between-imports comment
+import "google/protobuf/duration.proto"; // import-inline comment  2
+// between-package-and-import comment
+// import comment
+import "google/protobuf/timestamp.proto"; // import-inline comment
+
+// option comment 5
+option (custom.file_option) = true; // option-inline comment 5
+// option comment 6
+option (custom.file_thing_option) = {foo:1 bar:2 truth:false recursive:{foo:3 bar:4 truth:true}}; // option-inlinecomment 6
+// option comment 3
+option go_package = "foopb"; // option-inline comment 3
+// between-imports-and-options comment
+// option comment
+option java_multiple_files = true; // option-inline comment
+// option comment 4
+option java_outer_classname = "FooProto"; // option-inline comment 4
+// option comment 2
+option java_package = "com.foo"; // option-inline comment 2
+
+// leading comment one1
+// leading comment one2
+message One {
+  // message-option comment
+  // subsequent message-option comment
+  option (custom.message_option) = false; // message-option-inline comment
+  // message-option comment 2
+  option (custom.message_thing_option) = {foo:5}; // message-option-inline comment 2
+
+  // body comment 2
+  // body comment 3
+  // nested-message comment
+  message Three {
+    int64 three_one = 1;
+  }
+
+  /*
+  c-style comment
+  */
+  int64 one_one = 1 [(custom.field_option) = true];
+  Two one_two = 2 [(custom.field_option) = false, (custom.field_thing_option) = {foo:6}]; // field-option-inline comment
+  Three one_three = 3; /* field-inline c-style comment */
+  map<string,uint32> map_field = 4;
+  oneof oneof_field {
+    // oneof-field comment
+    option (custom.oneof_option) = true; // oneof-option-field-inline comment
+    option (custom.oneof_thing_option) = {foo:7}; // oneof-option-field-inline comment 2
+
+    int64 foo1 = 8;
+    string foo2 = 9;
+    // oneof-field comment 2
+    int64 woot3 = 10 [(custom.field_option) = false, (custom.field_thing_option) = {foo:8}]; // oneof-option-field-inline comment 3
+    // oneof-field comment 2
+    custom.Thing custom_field = 11; // oneof-option-field-inline comment 4
+  }
+  int64 integer_field = 12 [(custom.repeated_field_option) = 1, (custom.repeated_field_option) = 2];
+  int64 integer_field_two = 13 [(custom.field_thing_option) = {foo:9 repeated_foo:[1, 2]}, (custom.repeated_field_thing_option) = {foo:1 recursive:{foo:2 repeated_foo:[3, 4]}}, (custom.repeated_field_thing_option) = {foo:5 recursive:{foo:6 repeated_foo:[7, 8]}}, (custom.repeated_field_thing_option) = {foo:1 repeated_thing:[{foo:1 repeated_foo:[1, 2]}, {foo:3 repeated_foo:[3, 4]}]}];
```

Comment:
Formatting large compact options in one line probably isn't reasonable. I'll see what I can do to spread this across multiple lines when the options are long enough.